src/Microsoft.AspNetCore.Rewrite/Internal/ApacheModRewrite/RuleBuilder.cs

```diff
@@ -176,12 +176,6 @@ public void AddRule(string rule)
                 _actions.Add(new ChangeCookieAction(flag));
             }
 
-            if (flags.GetValue(FlagType.Env, out flag))
-            {
-                // parse env
-                _actions.Add(new ChangeEnvironmentAction(flag));
```

Comment:
It doesn't seem we throw this when other unsupported flags are used. We just silently ignore...should we open a separate issue for that?